docs/administration.platforms.md

```diff
@@ -0,0 +1,12 @@
+---
+id: administration.platforms
+title: Platform Notifications
+---
+
+Administrators can inform users about platform specific events by creating platform notifications:
+
+- **Platforms**: Select the platforms relevant to your notification so they can be shown on the affected platform's dashboards. Users can also [subscribe](meshcloud.profile.md#profile) to notifications by platform and will receive new notifications via email.
```

Comment:
d: you could also mention the the display of platform notifications for users on the docs page for my projects and/or project dashboard. DO we have a docs page for that atm?

Reply:
We don't have docs for project overview/dashboards: MS-1818